src/Compilers/CSharp/Portable/Symbols/NativeIntegerTypeSymbol.cs

```diff
@@ -46,6 +46,8 @@ internal NativeIntegerTypeSymbol(NamedTypeSymbol underlyingType) : base(underlyi
 
         public override IEnumerable<string> MemberNames => GetMembers().Select(m => m.Name);
 
+        internal override bool HasDeclaredRequiredMembers => false;
```

Comment:
We've gone back and forth on how much to support weirdly-defined well-known types. May be good to confirm with team.
Personally, I'm fine making this assumption.

Reply:
It seems like if this overload is removed, then we simply use the implementation from 'WrappedNamedTypeSymbol'. Was there a specific reason this overload was introduced?

Reply:
> We've gone back and forth on how much to support weirdly-defined well-known types. May be good to confirm with team.

> Was there a specific reason this overload was introduced?

`nint` and co are very, very special: they expose a limited set of properties from their underlying types, properties defined by the language. They are not defined to be required, so `false` is the only correct result to return from here.